unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarOnPointerDown/AvatarOutlineOnHoverEvent.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AvatarSystem;
+using DCL.Models;
+using DCLPlugins.UUIDEventComponentsPlugin.UUIDComponent.Interfaces;
+using UnityEngine;
+
+namespace DCL.Components
+{
+    /// <summary>
+    /// Complementary class to detect avatar outlining only,
+    /// It does not live on its own and should coexist with `AvatarOnPointerDown`
+    /// </summary>
+    public class AvatarOutlineOnHoverEvent : MonoBehaviour, IUnlockedCursorInputEvent, IPoolLifecycleHandler
```

Comment:
i dont really feel like naming this class as an Event, perhaps a handler

Reply:
I just maintain the current way of naming. More changes should be done in a separate refactoring